datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/HyperParameterValueTypeMapper.java

```diff
@@ -0,0 +1,37 @@
+package com.linkedin.datahub.graphql.types.mlmodel.mappers;
+
+import com.linkedin.datahub.graphql.generated.BooleanBox;
+import com.linkedin.datahub.graphql.generated.FloatBox;
+import com.linkedin.datahub.graphql.generated.HyperParameterValueType;
+import com.linkedin.datahub.graphql.generated.IntBox;
+import com.linkedin.datahub.graphql.generated.StringBox;
+import com.linkedin.datahub.graphql.types.mappers.ModelMapper;
+
+import lombok.NonNull;
+
+public class HyperParameterValueTypeMapper implements ModelMapper<com.linkedin.ml.metadata.HyperParameterValueType, HyperParameterValueType> {
+
+    public static final HyperParameterValueTypeMapper INSTANCE = new HyperParameterValueTypeMapper();
+
+    public static HyperParameterValueType map(@NonNull final com.linkedin.ml.metadata.HyperParameterValueType input) {
+        return INSTANCE.apply(input);
+    }
+
+    @Override
+    public HyperParameterValueType apply(@NonNull final com.linkedin.ml.metadata.HyperParameterValueType input) {
+        HyperParameterValueType result = null;
+
+        if (input.isString()) {
+            result = new StringBox(input.getString());
+        } else if (input.isBoolean()) {
+            result = new BooleanBox(input.getBoolean());
+        } else if (input.isInt()) {
+            result = new IntBox(input.getInt());
+        } else if (input.isDouble()) {
+            result = new FloatBox(input.getDouble());
```

Comment:
Both Float and Double type map to FloatBox?

Reply:
GraphQL does not have `Double` Type. So using Float in both the cases 
https://graphql.org/learn/schema/#scalar-types